services/web-app/pages/contributing/schema.mdx

```diff
@@ -535,3 +539,83 @@ For the value of the `navData` array, you can set the following keys.
 | `path`   | Path to the file.                                            | Required | String  | –       | <ul><li>Github URL according to the `https://{host}/{org}/{repo}/blob/{branch}/{path-to-file}` format </li><li>Path relative to the carbon.yml file location</li></ul> |
 | `items`  | Second level of navigation.                                  | Optional | Array   | –       | –                                                                                                                                                                      |
 | `hidden` | If set to true, the item will be hidden from the navigation. | Optional | Boolean | `false` | –                                                                                                                                                                      |
+
+## Schema references
+
+<AnchorLinks small>
+
+<AnchorLink>Absolute references</AnchorLink>
+<AnchorLink>Relative references</AnchorLink>
+<AnchorLink>Local references</AnchorLink>
+
+</AnchorLinks>
+
+For certain resource keys, platform allows the object to be defined as a reference to another
+existing resource instead of writing the actual value:
+
+```yml path="Example carbon.yml"
+designKits:
+  sample-design-kit:
+  $ref: './design-kits.yml#/designKits/sample-design-kit'
+```
+
+instead of:
+
+```yml path="Example carbon.yml"
+designKits:
+  sample-design-kit:
+    name: Sample Design Kit
+    tool: sketch
+    type: ui
+    status: stable
+    url: sketch://something
+    action: link
+```
+
+There are 3 ways you can define references in carbon: absolute, relative and local references.
+
+### Absolute references
+
+An absolute url that points to a resource contained in a yml file , e.g.:
+https://unpkg.com/@carbon-platform/resources/carbon.yml#/designKits/carbon-g10-sketch
+
+```yml path="Example carbon.yml"
+designKits:
+  sample-design-kit:
+  $ref: 'https://unpkg.com/@carbon-platform/resources/carbon.yml#/designKits/carbon-g10-sketch'
+```
+
+### Relative references
+
+A relative path to a resource contained in a yml file within the current repo, e.g.:
+'./design-kits.yml#/designKits/sample-design-kit'
```

Comment:
Should we use backticks instead of single quotes for these example values?

Reply:
Done!